sidebarsMemgraph.js

```diff
@@ -88,6 +88,12 @@ module.exports = {
       label: 'Database functionalities',
       items: ['database-functionalities/overview',
         'reference-guide/backup',
+        {
+          type: 'category',
+          label: 'Connecting to Kafka streams',
+          items: ['database-functionalities/streams/implement-transformation-module',
+            'database-functionalities/streams/kafka-streams'],
```

Comment:
Switch the order.

Reply:
I did them in this order, because jumping on the streams without transformations is not possible. 

What is the idea behind switching the order? First give a contextual overview, and then jump into the transformations? If so then it might be a better approach to create an overview page where we give a high level description of the whole streams-transformations things, and then jump into specifics.

Reply:
The transformation modules are not the "main" part I would say, just part of the whole streaming story. So you start describing how to manage streams, and when you come to the transformation part you say something along the lines of:
```
For the streams you need to define a transformation module. You can check here how to do it - LINK TO THE TRANSFORMATION EXPLANATION
```

The problem I see now is when the user opens `Connecting to streams` and the first thing he sees is a tutorial for some kind of transformations. What are they, how do they fit in with streams, he doesn't know anything at that point.
I wouldn't create an overview page just so we keep this part as simple as possible, more in the tutorial style where we just digress to the transformation part and then return to the `kafka-streams` page.